app/assets/stylesheets/application.scss

```diff
@@ -23,6 +23,10 @@ a:visited {
   color: $govuk-blue;
 }
 
+a[data-method='delete'] {
```

Comment:
Agreed. Just following the design. We can review at the end of the sprint with UCD.